app/app.go

```diff
@@ -113,6 +113,7 @@ import (
 	ibcclientclient "github.com/cosmos/ibc-go/v7/modules/core/02-client/client"
```

Comment:
@jelysn this PR is missing the setup handle logic to register the store key of the new module as well as the ParamKeyTable

Reply:
This module does not require any storage, let me check ParamKeyTable

Reply:
@jelysn any updates on that?

Reply:
Registered ParamKeyTable on upgrade handler - https://github.com/elys-network/elys/pull/195/files#diff-69701b9ad62c06683b76c4fdb5063d2005ef939af85e8a7191e872aad538ba06R87
Btw, it has logic to create store key and pass it even though it's not used - you mean to build upgrade handler on this PR?

Reply:
https://github.com/elys-network/elys/pull/195/files#diff-0f1d2976054440336a576d47a44a37b80cdf6701dd9113012bce0e3c425819b7R452
https://github.com/elys-network/elys/pull/195/files#diff-0f1d2976054440336a576d47a44a37b80cdf6701dd9113012bce0e3c425819b7R826 (Store Key)

Reply:
Yes the param key table needs to be set since 0.47 as long as you are using params in your module.

and corret the new module `transferhook` needs to be added to:

```
		storeUpgrades := storetypes.StoreUpgrades{
			Added: []string{
				clocktypes.ModuleName,
			},
		}
```

Reply:
It's added